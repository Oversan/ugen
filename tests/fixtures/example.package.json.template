{
  "name": "<% REPO_NAME %>",
  "version": "0.0.1",
  "description": "",
  "main": "application.js",
  "engines": {
    "npm": "^3.6.0",
    "node": "^5.6.0"
  },
  "repository": {
    "type": "git",
    "url": "<% GIT_URL %>"
  },
  "author": "",
  "license": "MTI",
  "bugs": {
    "url": "<% GIT_URL %>/issues"
  },
  "homepage": "http://<% REPO_NAME %>.example.com)"
}
