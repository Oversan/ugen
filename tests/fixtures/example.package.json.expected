{
  "name": "NewApp",
  "version": "0.0.1",
  "description": "",
  "main": "application.js",
  "engines": {
    "npm": "^3.6.0",
    "node": "^5.6.0"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/apporg/appname"
  },
  "author": "",
  "license": "MTI",
  "bugs": {
    "url": "https://github.com/apporg/appname/issues"
  },
  "homepage": "http://NewApp.example.com)"
}
